examples/widgets/histogram.html

```diff
@@ -119,7 +119,8 @@ <h4 class="as-subheader as-mb--12">
             bins: 10,
             start: 1000000,
             end: 127078679,
-            mode: 'REMOTE_FILTERED'
+            spatialFilter: 'viewport',
+            mode: 'precise'
```

Comment:
Probably the same here --> let's have a debug example with variants 'precise' / 'fast', but leave just the defaults for public examples